src/azure-cli/azure/cli/command_modules/container/custom.py

```diff
@@ -375,7 +375,6 @@ def _get_diagnostics_from_workspace(cli_ctx, log_analytics_workspace):
 # pylint: disable=unsupported-assignment-operation
 def _create_update_from_file(cli_ctx, resource_group_name, name, location, file, no_wait):
     resource_client = cf_resource(cli_ctx)
-    container_group_client = cf_container_groups(cli_ctx)
```

Comment:
Could you tell the purpose that we need remove this line?

Reply:
Previously, the `container_group_client` was only used to get the API version:
`container_group_client.api_version`

Since the `api_version` property was removed in the Track2 SDK, there was no reason to keep the `container_group_client` here.